DataFormats/BTauReco/interface/HiggsInteractionNetFeatures.h

```diff
@@ -0,0 +1,62 @@
+#ifndef DataFormats_BTauReco_HiggsInteractionNetFeatures_h
+#define DataFormats_BTauReco_HiggsInteractionNetFeatures_h
+
+#include <string>
+#include <vector>
+#include <unordered_map>
+#include "FWCore/Utilities/interface/Exception.h"
+
+namespace btagbtvdeep {
+
+  class HiggsInteractionNetFeatures {
```

Comment:
this looks identical to ```DeepBoostedJetFeatures```; please remove this class and move to ```DeepBoostedJetFeatures```

Reply:
Removed at 858b30df397. We also removed `HiggsInteractionNetTagInfos` because we could reuse `DeepBoostedJetTagInfos`. Let me know if that's appropriate.